test/e2e/service_bootstrap.py

```diff
@@ -87,21 +87,49 @@ def create_subnet(ec2, vpc_id: str, az: str, cidr: str) -> str:
     return subnet_id
 
 
+def create_service_linked_role(iam) -> str:
+    # TODO(jaypipes): There does not seem to be any way at all to list or
```

Comment:
In my previous experience of working on APIGateway team, SLR is created and managed by AWS services. Customers have nothing to do with it.
With that knowledge, I am wondering why do we need to create SLR ourselves.

API Gatewayv2 has the code path which created SLR in customer account and then used it to manage their resources.

Reply:
@vijtrip2 unfortunately, for ElasticsearchService, that is not the case...

If you don't set up the SLR yourself and associate your AWS account with the `es.amazonaws.com` SLR, the following happens and ends up in a TerminalCondition:

> Before you can proceed, you must enable a service-linked role to give Amazon ES permissions to access your VPC.

Apparently, this isn't a new problem :(

https://github.com/cloudposse/terraform-aws-elasticsearch/issues/5

Note that Terraform issue was filed 3 years ago.